.github/workflows/release.yaml

```diff
@@ -76,6 +76,7 @@ jobs:
             shell: bash
             run: |
               echo "Building Docker Images"
+              poetry install
```

Comment:
is the env being activated below?

Reply:
yes, after this it happens on the line `source env.sh || exit 1`